### ISSUE TITLE
LDDtool does not insert references to needed namespaces

### ISSUE THREAD

**Issue by msbentley** [NASA-PDS/pds4-information-model#303]
**Describe the bug**
In the PSA we use a cross-mission dictionary which, for example, declares a set of classes and attributes used by all missions. We then check/enforce the values of these attributes by (often rules-only) mission dictionaries.

Currently, building the two of these together does not result in correct namespace references.

For example, when building the BepiColombo dictionary I try:

`lddtool -lp Ingest_PDS4_PSA_1300.xml Ingest_PDS4_PSA_BC_1100.xml`

which results in several issues:

1. Wrong namespace in XSD
```
  <xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
    targetNamespace="http://psa.esa.int/psa/bc/v1mission/bc/v1"
    xmlns:bc="http://psa.esa.int/psa/bc/v1mission/bc/v1"
    xmlns:pds="http://pds.nasa.gov/pds4/pds/v1"
    elementFormDefault="qualified"
    attributeFormDefault="unqualified"
    version="1.1.0.0">
```
as seen something odd is happening with the namespace, which should be `http://psa.esa.int/psa/bc/v1` - an extra `mission/bc/v1` is being added.

2. Wrong namespace in SCH
`<sch:ns uri="http://pds.nasa.gov/pds4/mission/bc/v1" prefix="bc"/>`

3. No reference to the PSA namespace.

Items 1 and 2 are, I believe, covered by #301 so this ticket mainly focusses on the last.

Attached: [Ingest_PDS4_PSA.zip](https://github.com/NASA-PDS/pds4-information-model/files/5874679/Ingest_PDS4_PSA.zip) are a PSA and PSA mission dictionary, which can also be used for testing #301 and #302.

**Expected behavior**
Regarding item 3 - the PSA namespace should be inserted in the BepiColombo dictionary since elements of it are referenced. Currently we run `xmlstarlet` after lddtool to correct this.

**Version of Software Used**
```
LDDTool Version: 12.1.0
Built with IM Version: 1.15.0.0
Build Date: 2021-01-11 12:52:30
Configured IM Versions: [1F00, 1E00, 1D00, 1C00, 1B10, 1B00]
```

**Desktop (please complete the following information):**
 - OS: linux
 - Version: ubuntu 20.04

**Comment by jshughes** [comment]
Issue #304 should have addressed the issues specific to this issue, #303.

**Comment by jordanpadams** [comment]
@msbentley we think this is fixed with #304 fix. please verify

**Comment by msbentley** [comment]
Confirmed that the PSA namespace is now added to the output schematron in this case:
`<sch:ns uri="http://psa.esa.int/psa/v1" prefix="psa"/>`
I'll reply for the other issues in their respective tickets, thanks!
